frontend/src/components/project/create_task.jsx

```diff
@@ -40,7 +40,7 @@ class CreateTask extends React.Component {
           value={this.state.details} onChange={this.update('details')} />
         </label>
 
-      <button type="submit">Create Task!</button>
+        <button id="update-button" type="submit">Create Task!</button>
```

Comment:
I think your should use class rather than id to pick out elements for styling, if that's what this is for here